modules/azure-public-dns/dns-zones.auto.tfvars

```diff
@@ -0,0 +1,8 @@
+project   = "hmcts"
+component = "public-dns"
+location  = "uksouth"
+env       = "mgmt"
+
+zones = [
+  "test.example.com"
```

Comment:
we shouldn't be loading dummy data automatically, this should probably be empty by default?

also normally you would externalise configuration to a module, I would expect it to be passed in?

Reply:
You are correct there. What we've discussed with Piyush is that the plan output will be saved and passed in an an apply. Alternatively, the dns zone data can either be passed in or placed into ~/dns-records.auto.tfvars (preferably the former)